experimental/web_dashboard/lib/src/api/api.dart

```diff
@@ -2,44 +2,107 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:json_annotation/json_annotation.dart';
+
+part 'api.g.dart';
+
 /// Manipulates app data,
 abstract class DashboardApi {
-  ItemApi get items;
+  CategoryApi get categories;
   EntryApi get entries;
 }
 
-/// Manipulates [Item] data.
-abstract class ItemApi {
-  Future<Item> delete(String id);
-  Future<Item> get(String id);
-  Future<Item> insert(Item item);
-  Future<List<Item>> list();
-  Future<Item> update(Item item, String id);
-  Stream<List<Item>> allItemsStream();
-}
+/// Manipulates [Category] data.
+abstract class CategoryApi {
+  Future<Category> delete(String id);
 
-/// Something being tracked.
-class Item {
-  final String name;
-  String id;
+  Future<Category> get(String id);
+
+  Future<Category> insert(Category category);
+
+  Future<List<Category>> list();
```

Comment:
I think we can take out `latest` - `list` and `stream` are both required since `stream` doesn't return the initial value. (We could eventually switch these to a single RxDart stream to fix this.)

Reply:
removed `latest`